domains/instantnoodles.json

```
@@ -6,8 +6,7 @@
     "email": "info@InstantNoodles.is-a.dev"
   },
   "record": {
-    "A": ["172.66.47.94","172.66.44.162"],
+    "A": ["172.66.47.94", "172.66.44.162"],
     "MX": ["mx1.improvmx.com","mx2.improvmx.com"],
```

Trailing commas are not valid in JSON.
```suggestion
    "MX": ["mx1.improvmx.com","mx2.improvmx.com"]
```